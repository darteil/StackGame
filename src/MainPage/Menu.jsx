import React from 'react';
import { NavLink } from 'react-router-dom';
import styles from './styles.css';


const Menu = () => (
  <div className={styles.menu}>
    <h1>Stack</h1>
    <ul>
      <NavLink exact to={process.env.SUB_ROUTE || '/'} activeClassName={styles.active}>
        <li>
          Game
        </li>
      </NavLink>
      <NavLink to={`${process.env.SUB_ROUTE}/list-of-records` || '/list-of-records`'} activeClassName={styles.active}>
        <li>
          List of records
        </li>
      </NavLink>
    </ul>
  </div>
);

export default Menu;
