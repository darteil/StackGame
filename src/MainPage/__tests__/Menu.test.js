import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Menu from 'App/MainPage/Menu';

describe('Menu', () => {
  it('should render correctly', () => {
    const output = shallow(<Menu />);
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
