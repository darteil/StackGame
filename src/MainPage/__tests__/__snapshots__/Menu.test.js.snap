// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu should render correctly 1`] = `
<div>
  <h1>
    Stack
  </h1>
  <ul>
    <NavLink
      activeClassName="active"
      aria-current="page"
      exact={true}
      to="/"
    >
      <li>
        Game
      </li>
    </NavLink>
    <NavLink
      activeClassName="active"
      aria-current="page"
      to="/list-of-records"
    >
      <li>
        List of records
      </li>
    </NavLink>
  </ul>
</div>
`;
